variables.tf

```diff
@@ -230,6 +230,12 @@ variable "primary_vnic_nsg_ids" {
   default     = null
 }
 
+variable "instance_launch_options_network_type" {
+  description = "Emulation type for the physical network interface card (NIC). Accepted values are E1000, VFIO or PARAVIRTUALIZED"
+  type        = string
+  default     = "PARAVIRTUALIZED"
+}
+
```

Comment:
it could be helpful to add validation control to limit input.

es: 

```HCL
validation {
    condition     = contains(["E1000", "VFIO", "PARAVIRTUALIZED"], var.instance_launch_options_network_type)
    error_message = "Accepted values for var: instance_launch_options_network_type are (E1000, VFIO, PARAVIRTUALIZED)"
  } 
```